docs/modules/ROOT/pages/explanations/decisions/redis_galera.adoc

```diff
@@ -0,0 +1,64 @@
+= Redis and MariaDB/Galera Provisioning
+
+This page combines the decisions for deploying Redis, MariaDB and Galera, as they are already in use for SPKS.
+
+== Problem
+
+We want to re-use proven methods to deploy Redis and MariaDB/Galera.
+
+== Operators
+
+For Redis as well as MariaDB/Galera there are a few operators available.
+But most of those are in a very early development stage.
+Or don't support the applications we want to provide.
+
+The following section contains the operators that have been looked at.
+We might re-evaluate them in the future.
+
+=== Redis
+
+* https://ibm.github.io/operator-for-redis-cluster/
+** Very alpha
```

Comment:
a question here: why is this operator very alpha? I think we talked about it in the refinement, but now I don't recall it and I just saw the last tag is `v1.2.3`, so I wanted to ask.

Reply:
I think you're looking at the wrong one, this one is at 0.3.13.

Reply:
oh, right, sorry. All clear, thanks